app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = [Person(person["name"], person["age"]) for person in people]
+
+    for name_wife in people:
+        if name_wife.get("wife") is not None:
```

Comment:
There is no need to check this condition with `is not None`. Besides that, you could allocate a wife`s name in a variable and simplify the internals of the if condition. Take into account, that you should not place arguments on the same line with `]` in case `[` does not have arguments on the same line
```suggestion
        if wife_name := name_wife.get("wife"):
```